pyfr/backends/cuda/cublaslt.py

```diff
@@ -0,0 +1,296 @@
+from collections import namedtuple
+from ctypes import (POINTER, Structure, byref, c_double, c_float, c_int,
+                    c_int64, c_size_t, c_uint64, c_void_p, sizeof)
+
+import numpy as np
+
+from pyfr.backends.cuda.driver import CUDADevAlloc
+from pyfr.backends.cuda.provider import CUDAKernel, CUDAKernelProvider
+from pyfr.ctypesutil import LibWrapper
+
+
+# Possible CUBLAS exception types
```

Comment:
LT

Reply:
Yep, they stylise as CUBLASLt, so that's what I've done

Reply:
Comment says CUBLAS though